### PR TITLE
Update E2E tests

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #146

### CODE REVIEW

File: test/panel.spec.ts [hunk 1]
```diff
@@ -1,49 +1,164 @@
...
+       * Go To Panels dashboard urls.json
+       * return dashboardPage
+       */
+      const dashboard = await readProvisionedDashboard({ fileName: 'urls.json' });
+      const dashboardPage = await gotoDashboardPage({ uid: dashboard.uid });
+
+      /**
+       * Panel loading...
+       */
+      await page.waitForTimeout(1500);
```
Comment (asimonok) [inline test/panel.spec.ts:129]: please check if panel loaded in helper via checking if loading message disappeared

**Review by asimonok — CHANGES_REQUESTED** [review]

File: test/panel.spec.ts (hunk 1 above)
Comment (vitPinchuk) [reply test/panel.spec.ts]: `page.waitForTimeout` was removed

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**